### PR TITLE
[XeTLA] Add xetla splitk gemm

### PR DESCRIPTION
**Pull request by ESI-SYD**

### CODE REVIEW

File: benchmarks/triton_kernels_benchmark/gemm_splitk_benchmark.py [hunk 1]
```diff
@@ -148,23 +149,36 @@ def benchmark(M, N, K, provider):
...
+        c = torch.empty((M, N), device='xpu', dtype=torch.float32)
+        acc = torch.empty((M, N), device='xpu', dtype=torch.float32)
+        cnt = torch.empty((M, N), device='xpu', dtype=torch.int32)
+
+        name = f'gemm_splitk_shape_{M}_{K}_{N}'
+        func = getattr(xetla_kernel, name)
+        xetla_fn = lambda: func(a, b, c, acc, cnt)
+        torch_fn = lambda: torch.matmul(a, b).to(torch.float32)
+
+        # benchmark_suit.assert_close(xetla_fn(), torch_fn(), atol=1e-4, rtol=1.0, err_msg='xetla to torch')
```
Comment (chengjunlu) [inline benchmarks/triton_kernels_benchmark/gemm_splitk_benchmark.py:172]: Can we enable the accuracy checking?

Comment (ESI-SYD) [reply benchmarks/triton_kernels_benchmark/gemm_splitk_benchmark.py:172]: XeTLA still have precision issue to torch.

**Review by anmyachev — APPROVED** [review]
LGTM! @chengjunlu?

**Review by whitneywhtsang — APPROVED** [review]

**Review by chengjunlu — APPROVED** [review]

**Comment by chengjunlu** [comment]
Let's focus on the performance of the XeTLA kernel.

**Pull request merged**